docs/workflow.rst

```diff
@@ -36,9 +36,7 @@ Datamodel.py
 
 Before running the BEAST, you will need to modify this file to specify
 the required parameters for generating models and fitting data.
-These parameters are described in the beast `setup documentation <beast_setup.rst>`_.
-
-
+These parameters are described in the beast :ref:`setup documentation <beast_setup_datamodel>`.
```

Comment:
Unfortunately, this works when building the documentation with Sphinx, but not on GitHub.

Reply:
Is the new code correct for sphinx?  If so, then that is what we want.  The docs of interest are those that are at readthedocs, not in the github repository docs subdirectory.  Ok?